frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,12 +99,29 @@ const generateTableHeaders = (
     canAddOrDeletePolicy?: boolean;
     tableType?: string;
   },
-
+  policiesList: IPolicyStats[] = [],
```

Comment:
I don't think we pass the table data in to generating columns like this anywhere else, but don't see why not. Maybe worth a look by @ghernandez345 or @gillespi314?

Reply:
Am I understanding the thinking here is to iterate over the list with `some` until a record with an updated at timestamp is found because some records may have null-ish values for `host_count_updated_at`?

An alternative approach might be to use `find` with optional chaining and then use the `date-fns` library that the frontend uses elsewhere to do the date formatting. (We could skip the `find` and just use the first element of the list if we don't have to worry about null-ish `host_count_updated_at` values.)

```ts
  // find first policy item with host_count_updated_at
  const updatedAt =
    policiesList.find((p) => !!p.host_count_updated_at)
      ?.host_count_updated_at || "";

  let timeSinceHostCountUpdate = "";
  if (updatedAt) {
    try {
      timeSinceHostCountUpdate = formatDistanceToNowStrict(new Date(updatedAt), { addSuffix: true });
    } catch (e) {
      // TODO: handle exception or do nothing
    }
  }
```

I'm on the fence about passing the policies list to the generate function. We should definitely keep a close eye on it.  The columns need to be memoized properly in order for react-table to be performant and minimize expensive render cycles.

Reply:
I went with @gillespi314's suggested implementation. Thank you.